http.go

```diff
@@ -42,19 +42,19 @@ const (
 // non validating client depending on the value of verify.
 func GetHTTPClient(verify SSLHostnameVerification, certs ...string) *http.Client {
 	if len(certs) > 0 {
-		return GetHTTPClientWithCerts(verify, certs)
+		return getHTTPClientWithCerts(verify, certs)
 	}
 	if verify == VerifySSLHostnames {
 		return GetValidatingHTTPClient()
```

Comment:
Since you are making changes here, I have the same question about `GetValidatingHTTPClient` and `GetNonValidatingHTTPClient`. Are we invoking these functions outside of this package? If not, we can probably also make them internal and have `GetHTTPClient` as the entry point.

Reply:
They are both directly used by juju/juju